packages/sync/src/modules/class-module.php

```diff
@@ -273,6 +273,7 @@ protected function enqueue_all_ids_as_action( $action_name, $table_name, $id_fie
 	 * @return array|object|null
 	 */
 	public function get_next_chunk( $config, $status, $chunk_size ) {
+		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
```

Comment:
@mdbitz I'd love your thoughts on confirming this is acceptable.

Reply:
We should ideally switch this to a prepared sql vs what this is. Creating an issue to track it, will tackle it in the short term, but shouldn't block this commit to avoid phpcs warnings.